NorthstarDLL/httprequesthandler.cpp

```diff
@@ -0,0 +1,504 @@
+#include "pch.h"
+#include "httprequesthandler.h"
+#include "version.h"
+#include "squirrel.h"
+#include "tier0.h"
+
+HttpRequestHandler* g_httpRequestHandler;
+
+void HttpRequestHandler::StartHttpRequestHandler()
+{
+	if (IsRunning())
+	{
+		spdlog::warn("HttpRequestHandler::StartHttpRequestHandler was called while IsRunning() is true");
```

Comment:
With MSVC you can use the \_\_FUNCTION\_\_ macro to create a string literal of the function name at compile time, this will make the code more maintainable. If you change the name of the function, the macro takes care of it.

Reply:
Resolved on new branch, will re-make PR.